tests/unit/test_handlers.py

```diff
@@ -804,3 +807,77 @@ def test_add_md5_with_bytes_object(self):
         self.assertEqual(
             request_dict['headers']['Content-MD5'],
             'OFj2IjCsPJFfMAxmQxLGPw==')
+
+
+class TestParameterAlias(unittest.TestCase):
```

Comment:
What should be the desired behavior? We remove the alias but use the original's value or remove the alias and use the alias's value. Currently because of how its written the alias will clobber the original. Do you think an error should be thrown instead for this scenario saying the parameters provided is ambiguous?